modules/service/src/main/scala/lucuma/odb/data/StepExecutionState.scala

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.odb.data
+
+import lucuma.core.util.Enumerated
+
+/**
+ * The execution state of a recorded step.
+ *
+ * @param tag database tag
```

Comment:
There isn't actually an enumeration in the database for this, at least not yet.  It is used though in Json codecs.